shuttle/templates/schedules/shuttle_driver_check_in.html

```diff
@@ -29,33 +30,113 @@ <h4>Arriving at:</h4>
             <div class="submit">
-                <input class="btn btn-primary btn-sm" type="submit" value="Check out for break">
+                <input class="btn btn-primary btn-sm check-out-btn" type="submit" value="Check out for break">
             </div>
         </div>
         <div class="align-side-by-side">
             <div class="submit">
-                <input class="btn btn-primary btn-sm" type="submit" value="Check in from break">
+                <input class="btn btn-primary btn-sm check-in-btn" type="submit" value="Check in from break">
             </div>
         </div>
     </div>
     <div class="center-page">
-       <h5>Current status:<span class="badge badge-light">*Insert "on break"/"working"*</span></h5>
+       <h4>Current status:<span class="badge badge-light">Not on break</span></h4>
     </div>
 
-{% endblock %}
\ No newline at end of file
+    <script>
+    $(document).ready(function() {
+        $(".leaving-drop a").click(function () {
+            // Dropdown shows leaving location selected
+            $(".leaving-menu:first-child").text($(this).text());
+            $(".leaving-menu:first-child").val($(this).text());
+        });
+        $(".arriving-drop a").click(function () {
+            // Dropdown shows arriving location selected
+            $(".arriving-menu:first-child").text($(this).text());
+            $(".arriving-menu:first-child").val($(this).text());
+        });
+
+        $('.submit-leaving').click(function () {
+            // Sends selected location and user info to database.
+            var data = {"location": $('.leaving-menu').val(), "direction": "departure"}
+            $.ajax({
+                type: 'POST',
+                contentType: 'application/json',
+                url: "{{ url_for('SchedulesView:send_driver_check_in_info') }}",
+                data: JSON.stringify((data)),
+            }).done(function (data) {
+                alert(data);
+            })
+                .fail(function () {
+                    alert("Something went wrong. Please try again or call the ITS Help Desk at 651-638-6500");
+                });
+        });
+        $('.submit-arriving').click(function () {
+            // Sends selected location and user info to database.
+            var data = {"location": $('.arriving-menu').val(), "direction": "arrival"}
+            $.ajax({
+                type: 'POST',
+                contentType: 'application/json',
+                url: "{{ url_for('SchedulesView:send_driver_check_in_info') }}",
+                data: JSON.stringify((data)),
+            }).done(function (data) {
+                alert(data);
+            })
+                .fail(function () {
+                    alert("Something went wrong. Please try again or call the ITS Help Desk at 651-638-6500");
+                });
+        });
+
+        $('.check-out-btn').click(function () {
+            // Sends break info to database
+            var data = {"break": "Y"}
```

Comment:
Sometimes you capitalize break and other times you don't. Please standardize it.